daffodil-core/src/main/scala/edu/illinois/ncsa/daffodil/dsom/GroupBase.scala

```diff
@@ -32,28 +32,26 @@
 
 package edu.illinois.ncsa.daffodil.dsom
 
-import scala.xml.Node
-import scala.xml._
 import edu.illinois.ncsa.daffodil.schema.annotation.props.AlignmentType
 import edu.illinois.ncsa.daffodil.schema.annotation.props.gen.AlignmentUnits
 import java.lang.{ Integer => JInt }
 
-abstract class GroupBase(xmlArg: Node, parentArg: SchemaComponent, position: Int)
-  extends Term(xmlArg, parentArg, position) {
+trait GroupBase
```

Comment:
GroupBase and ModelGroup can be collapsed together now. The right name for the merged trait/class is ModelGroupTerm. Similarly the right name for ElementBase is probably ElementTerm.

Reply:
GroupBase class elminated, but postponing the renames ModelGroup and ElementBase